src/core/ext/filters/client_channel/backup_poller.cc

```diff
@@ -95,6 +95,8 @@ static void g_poller_unref(grpc_exec_ctx* exec_ctx) {
                                             p, grpc_schedule_on_exec_ctx));
     gpr_mu_unlock(p->pollset_mu);
     grpc_timer_cancel(exec_ctx, &p->polling_timer);
+  } else {
+    gpr_mu_unlock(&g_poller_mu);
```

Comment:
wouldn't you have to unconditionally unlock?

Reply:
It leaves the old unlock within the `if`, I think it's ok

Reply:
Ah yes.